nautobot/docs/user-guide/core-data-model/extras/contact.md

```diff
@@ -3,3 +3,18 @@
 A contact represents an individual that can be associated with other model instances. It stores the necessary information to uniquely identify and contact the individual. The information that can be stored on a contact includes the individual's name, phone number, email, and address. When a contact is assigned to an object, the status of such assignment needs to be specified.
 
 A contact can be assigned to one or more [teams](team.md)
+
++/- 2.2.1
+
+    A model can opt out of the ability to be asssociated with contacts or teams by specifying `is_contact_associable = False` in the model class definition.
+
+```python
+
+class YourModel(PrimaryModel):
+    name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, db_index=True)
+    is_contact_associable_model = False
+
+    class Meta:
+        abstract = True
+
+```
```

Comment:
Possibly indent this block by four spaces so that it's included inside the 2.2.1 panel?